README.adoc

```diff
@@ -111,44 +116,44 @@ Add a simple CORS configuration to the [hotspot file=0]`server.xml` file:
 `src/main/liberty/config/server.xml`
 ----
 server.xml
-[source, xml, linenums, role='code_column']
+[source, xml, linenums, role='code_column hide_tags=preflight-config']
 ----
-include::finish/src/main/liberty/config/server.xml[tags=**;!preflight-config]
+include::finish/src/main/liberty/config/server.xml[]
 ----
 
-The simple CORS configuration contains the following attributes:
+The CORS configuration contains the following attributes:
 
 |===
 | *Configuration Attribute* | *Value*
-|[hotspot=12 file=0]`domain` | The endpoint to be configured for CORS requests. The value is set to `/configurations/simple`.
-|[hotspot=13 file=0]`allowedOrigins` | Origins that are allowed to access the endpoint. The value is set to `openliberty.io`.
-|[hotspot=14 file=0]`allowedMethods` | HTTP methods that a client is allowed to use when it makes requests to the endpoint. The value is set to `GET`.
-|[hotspot=15 file=0]`allowCredentials` | A boolean that indicates whether the user credentials can be included in the request. The value is set to `true`.
-|[hotspot=16 file=0]`exposeHeaders` | Headers that are safe to expose to clients. The value is set to `MyHeader`.
+|[hotspot=16 file=0]`domain` | The endpoint to be configured for CORS requests. The value is set to `/configurations/simple`.
+|[hotspot=17 file=0]`allowedOrigins` | Origins that are allowed to access the endpoint. The value is set to `openliberty.io`.
+|[hotspot=18 file=0]`allowedMethods` | HTTP methods that a client is allowed to use when it makes requests to the endpoint. The value is set to `GET`.
+|[hotspot=19 file=0]`allowCredentials` | A boolean that indicates whether the user credentials can be included in the request. The value is set to `true`.
+|[hotspot=20 file=0]`exposeHeaders` | Headers that are safe to expose to clients. The value is set to `MyHeader`.
 |===
 
 Save the changes to the [hotspot file=0]`server.xml` file. The `/configurations/simple` endpoint is now ready to be
 tested with a simple CORS request.
 
-Open the [hotspot file=1]`TestCors.java` file in the `src/test/java/it/io/openliberty/guides/cors/` directory.
+The Open Liberty server was started in development mode at the beginning of the guide and all the changes were automatically picked up.
 
-Now we will test the simple CORS configuration that you added. Add the [hotspot=25-33 file=1]`testSimpleCorsRequest` method to the `TestCors` class.
+Now we will test the simple CORS configuration that you added. Add the [hotspot=testSimpleCorsRequest file=1]`testSimpleCorsRequest` method to the `CorsIT` class.
```

Comment:
Change
"Now we will test the simple CORS configuration that you added."
To
"Now, test the simple CORS configuration that you added."

Reply:
Updated